style.css

```diff
@@ -1,3 +1,83 @@
-.color-blue {
-  color: blue;
+* {
+  font-family: "Poppins", sans-serif;
+  color: ash;
+}
+
```

Comment:
- [OPTIONAL] In order to use those special fonts like `Poppins, Roboto. etc..`, you must, first of all, import them from [google fonts](https://fonts.google.com/) so you can be able to use them on your text and see the changes on the pages. I highly recommend you import them first and then use them because at this moment your fonts are not changing anything on the page. Kindly fix that.